src/comm/TCPLink.cc

```diff
@@ -81,6 +81,7 @@ void TCPLink::_writeBytes(const QByteArray data)
 
     if (_socket) {
         _socket->write(data);
+        emit bytesSent(this, data);
```

Comment:
I only see TCP and UDP adding the bytesSent signal. What about SerialLink and BLuetoohLink?